texttunnel/chat.py

```diff
@@ -1,8 +1,11 @@
-from typing import Dict, List
-from dataclasses import dataclass
-from jsonschema import Draft7Validator, exceptions
-import tiktoken
 import json
+from typing import Any, Callable, Dict, List, Optional
+
+import tiktoken
+from jsonschema import Draft7Validator, exceptions
+
+from texttunnel import utils
+from texttunnel.models import Model
 
```

Comment:
Not at this location, but: line 40 has a `count_tokens` function. But we also have that in the `utils.py` script. Is this still necessary..?

Reply:
It's good that you noticed, no it's not necessary. Removed it